code/modules/admin/verbs/adminhelp.dm

```diff
@@ -91,24 +91,47 @@
 	if(src.handle_spam_prevention(msg,MUTE_ADMINHELP))
 		return
 
+
+	var/datum/adminticket/ticket
+	var/ref_client = "\ref[src]"
+	for(var/I in admintickets)
+		var/datum/adminticket/T = I
+		if(T.permckey == src.ckey && T.resolved == "No")
+			if(alert(usr,"You already have an adminhelp open, would you like to bump it?", "Bump Adminhelp", "Yes", "No") == "Yes")
+				T.logs += "[src.ckey] has bumped this adminhelp!"

```

Comment:
Please use `LAZY_ADD` to add things into lists

More about that can be found [here](https://github.com/HippieStation/HippieStation13/blob/master/code/__HELPERS/lists.dm#L459)